auth/server/src/main/java/com/seagates3/controller/UserPolicyController.java

```diff
@@ -111,6 +113,38 @@ ServerResponse detach() throws DataAccessException {
     return serverResponse;
   }
 
+ public
+  ServerResponse list() throws DataAccessException {
+    String pathPrefix = requestBody.get("PathPrefix");
```

Comment:
use it a constant

Reply:
This will be done once https://github.com/Seagate/cortx-s3server/pull/1651 is merged.